src/main/java/games/strategy/engine/lobby/server/userDB/HashedPassword.java

```diff
@@ -0,0 +1,20 @@
+package games.strategy.engine.lobby.server.userDB;
+
+import com.google.common.base.Strings;
+import games.strategy.util.MD5Crypt;
```

Comment:
Checkstyle: "'games.strategy.util.MD5Crypt' should be separated from previous import group by one line."